Revoke-Obfuscation.psm1

```diff
@@ -999,6 +999,10 @@ C:\PS> Get-WinEvent -LogName "Microsoft-Windows-PowerShell/Operational" | Get-Rv
 
 C:\PS> Get-CSEventLogEntry -LogName Microsoft-Windows-PowerShell/Operational | Get-RvoScriptBlock
 
+.EXAMPLE
+
+C:\PS> Get-RvoScriptBlock -Helix (tap search "class=ms_windows_powershell eventid=4104" InstanceID | ConvertFrom-Json)
```

Comment:
![image](https://user-images.githubusercontent.com/11475352/37739575-c96dc656-2d17-11e8-8115-d2eab5d197bd.png)

This example doesn't appear to work.

Reply:
I don't have familiarity with the Helix API so that wasn't being facetious :) Is the result of a Helix search a bunch of JSON? If so, it might be better to add a bit more description to the the example (like you did for the 'Helix' parameter below)

Reply:
You are correct, 'TAP' is a python based API helper for Helix. It returns JSON objects, which need to be converted to PSCustomObject using ConvertFrom-JSON. Since not everyone has this wrapper, I've changed the example to use Invoke-Webrequest and give an example in the .Synopsis.Notes section.